pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/ServerReloadControllerJobStatusResponse.java

```diff
@@ -20,15 +20,25 @@
 
 import java.util.Map;
 
+
 public class ServerReloadControllerJobStatusResponse {
   private double _timeElapsedInMinutes;
   private double _estimatedTimeRemainingInMinutes;
   private int _totalSegmentCount;
+  private int _totalCount;
```

Comment:
What does total count stands for? Seems you use it for both segment count and server count, but we already have stats for both of them

Reply:
So this class was used for reloading all segments API - wanted to share the same response class here and tried to generify this. Progress is successCount / totalCount (didn't want to overload totalSegmentCount for reloadTable as that's very specific to segments and not table as such).

The progress for this is how many servers have reloaded their table.

Reply:
Since the target is table, which is always single, I don't think this response should be reused. If we want to reuse this, then we need to track the segments already loaded in the new table data manager